frontend/ui/src/pages/Inbox/ConversationsFilter/index.tsx

```diff
@@ -111,7 +111,7 @@ const ConversationsFilter = (props: ConversationsFilterProps) => {
     if (filteredPaginationData.loaded) {
       return (
         <div className={styles.filterCount}>
-          {`Filtered: ${filteredPaginationData.filtered_total} Total: ${filteredPaginationData.total}`}
+          {`Filtered: ${filteredPaginationData.filteredTotal} Total: ${filteredPaginationData.total}`}
```

Comment:
fixed this snake case in the UI, I am not so sure why this was not in camelCase...